patches/server/0933-Added-byte-array-serialization-deserialization-for-P.patch

```diff
@@ -0,0 +1,41 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Nex <nex@bits.team>
+Date: Thu, 24 Feb 2022 16:28:07 +0100
+Subject: [PATCH] Added byte array serialization/deserialization for
+ PersistentDataContainers
+
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/persistence/CraftPersistentDataContainer.java b/src/main/java/org/bukkit/craftbukkit/persistence/CraftPersistentDataContainer.java
+index f0588bec9be09cb8273c310fb3de8bfe72dee9e5..d85d037dae05f31a16309ea3065b61faa10e65ff 100644
+--- a/src/main/java/org/bukkit/craftbukkit/persistence/CraftPersistentDataContainer.java
++++ b/src/main/java/org/bukkit/craftbukkit/persistence/CraftPersistentDataContainer.java
+@@ -168,5 +168,29 @@ public class CraftPersistentDataContainer implements PersistentDataContainer {
+ 
+         return this.customDataTags.containsKey(key.toString());
+     }
++
++    @Override
++    public byte[] serializeToBytes() throws java.io.IOException {
++        net.minecraft.nbt.CompoundTag root = this.toTagCompound();
++        try (java.io.ByteArrayOutputStream byteArrayOutput = new java.io.ByteArrayOutputStream()) {
++            try (java.io.DataOutputStream dataOutput = new java.io.DataOutputStream(byteArrayOutput)) {
```

Comment:
You don't need to nest two try with block, java will close all the parent stream from the lowest stream which mean when the data output stream will be close the byte array one will be also automatically close, so no need to close two time the byte stream